Cargo.toml

```diff
@@ -109,19 +115,24 @@ rlp = { version = "0.5.1", default-features = false }
 substrate-bn = { version = "0.6.0", default-features = false }
 
 # IPLD/Encoding
-cid = { version = "0.10.1", default-features = false, features = ["serde-codec", "std"] }
-multihash = { version = "0.18.1", default-features = false , features = ["multihash-impl"]}
+cid = { version = "0.10.1", default-features = false, features = [
+     "serde-codec",
+     "std",
+] }
+multihash = { version = "0.18.1", default-features = false, features = [
+     "multihash-impl",
+] }
 libipld-core = { version = "0.13.1", features = ["serde-codec"] }
 integer-encoding = { version = "3.0.3", default-features = false }
 
 # helix-onchain
-fvm_actor_utils = "9.0.0"
-frc42_dispatch = "5.0.0"
-frc46_token = "9.0.0"
+fvm_actor_utils = "10.0.0"
+frc42_dispatch = "6.0.0"
+frc46_token = "10.0.0"
```

Comment:
Please make these and other dependency updates against `main` as a separate PR, that you can create and land now. Remove this noise from the semantic changes introduced in this PR.